src/server/middleware/api/index.js

```diff
@@ -2,10 +2,11 @@ const routes = require('express').Router()
 
 const comments = require('./comments')
 const discourse = require('./discourse')
-const github = require('./github')
+const { issues, stars } = require('./github')
 
 routes.get('/comments', comments)
 routes.get('/discourse', discourse)
-routes.get('/github', github)
+routes.get('/github', issues)
```

Comment:
do we want to change the route to `/github/issues`?

Reply:
I kept it the same for backward compatibility reasons, but it would make sense to change it and I can do it.

Reply:
Changed!